src/main/resources/assets/lint/lang/en_us.json

```diff
@@ -110,6 +110,8 @@
   "item.lint.suspicious_looking_disc": "Suspicious Looking Disc",
   "item.lint.tarscan_shard": "Tarscan Shard",
   "item.lint.tater_essence": "Tater Essence",
+  "item.lint.guide_book": "Lint Guide book",
+  "item.lint.guide_book.landing_text": "TODO",
```

Comment:
TODO

Reply:
i dont know what to put there. commit a change if you think you know

Reply:
@valoeghese got any ideas?